bin/protogen

```diff
@@ -50,6 +50,7 @@ def main():
     # 3. Generate PoET protos
     proto_dir = JOIN(TOP_DIR, "poet/sawtooth_poet/protos")
     protoc(proto_dir, "poet", "sawtooth_poet/protobuf")
+    protoc(proto_dir, "validator", "sawtooth_validator/protobuf")
```

Comment:
You can put this into `sawtooth_poet/protobuf` as well, so we can avoid an possible `PYTHONPATH` conflicts